src/locale/all.json

```diff
@@ -0,0 +1,22 @@
+{
+  "SNPivotTable_LimitedData": {
```

Comment:
Can you delete this file?

Reply:
I actually did, but weird, let me check it again